addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-linux-instance.cfn.yml

```diff
@@ -93,6 +105,7 @@ Resources:
                 - 'ec2.amazonaws.com'
             Action:
               - 'sts:AssumeRole'
+      ManagedPolicyArns: ['arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore']
```

Comment:
Could you take a look at this managed policy and confirm there's no more scope of constricting the permission boundary? For example, if this policy only gets SSM resources then there's no reason to allow all actions in permission boundary.